htdocs/static/js/src/webfront.js

```diff
@@ -301,6 +301,33 @@ require([
 
 
     /**
+     * Import dashboard sumbit function
+     */
+    function setupImportDashboard() {
+        $('#dashboard-import form').submit(function(event) {
+            event.preventDefault();
+            var formData = new FormData($(this)[0]);
+
+            $.ajax({
+                url: $(this).attr("action"),
+                type: 'POST',
+                data: formData,
+                processData: false,
+                contentType: false,
+                success: function (data) {
+                    if (data.location) {
+                        window.location = data.location;
+                    }
+                    if (data.error) {
```

Comment:
Er det noen grunn til at vi returnerer 200 men sjekker for data.error i stedet for riktig status og en fail-callback? Er vi inspirert av Go? =)